feature-detects/css/nthchild.js

```diff
@@ -27,15 +27,12 @@ define(['Modernizr', 'testStyles'], function(Modernizr, testStyles) {
   // A Javascript loop then tests if the `<div>`s have the expected width
   // using the modulus operator.
   testStyles('#modernizr div {width:1px} #modernizr div:nth-child(2n) {width:2px;}', function(elem) {
-    Modernizr.addTest('nthchild', function() {
-      var elems = elem.getElementsByTagName('div'),
-      test = true;
+    var elems = elem.getElementsByTagName('div'),
+    correctWidths = true;
```

Comment:
Can you change this to multiple `var`'s this is an old one but all other tests should use this pattern going forward